.github/workflows/buildandtest.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [ "main" ]
   pull_request:
```

Comment:
Was the buildandtest action previously not run on PRs?

Reply:
it only ran on PRs targeted to main, this allows stacked PRs